lib/user_approval.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module UserApproval
```

Comment:
@peggles2 , I realize this module was refactored out of the rake task, but it would be preferable to have specs for this module directly. Right now this code is only tested indirectly via the rake task. I'll approve for now, but consider adding a module spec when refactoring.

Reply:
Yeah I thought about that. If the methods in the module fails the specs will fail when it accesses the module. Nevertheless I am not apposed to more tests so sure we can do that.

Reply:
👍